docker-compose.yml

```diff
@@ -14,17 +42,25 @@ services:
         condition: service_healthy
     expose:
       - "${FRONTEND_PORT}"
+    restart: always
+    security_opt:
+      - "no-new-privileges=true"
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "10m"
+        max-file: "3"
 
   jsonrpc:
     build:
       context: ./
       dockerfile: ./docker/Dockerfile.backend
-      target: debug
+      target: ${BACKEND_BUILD_TARGET:-prod}
     environment:
       - FLASK_SERVER_PORT=${RPCPORT}
-      # TODO: remove this in production
-      - PYTHONUNBUFFERED=1
-      - RPCDEBUGPORT=${RPCDEBUGPORT}
+      # # TODO: remove this in production
+      # - PYTHONUNBUFFERED=1
+      # - RPCDEBUGPORT=${RPCDEBUGPORT}
```

Comment:
we should probably use an envvar to set/unset debug mode. As it is now, you've blocked us on development. Please change

Reply:
I've added `SERVER_NAME` env var